### PR TITLE
return error when node value evaluation is ambiguous

### LINKED ISSUE #38: [lang] add error message when both existing yaml node value and computed value are present?
Input:
```
x: #@ { "first": "is included"}
  second: is ignored
```
The second line is silently ignored.

Also here:

```
--- #@ { "first": "is included"}
this line: is ignored too
```

If this is not valid, I'd expect an error message instead of silently ignoring part of the input.

### PR DESCRIPTION
**Pull request by nimakaviani**
fixes #38

### CODE REVIEW

File: pkg/yamltemplate/metas.go [hunk 1]
```diff
@@ -61,6 +61,13 @@ func NewMetas(node yamlmeta.Node, opts MetasOpts) (Metas, error) {
 
 				if node.GetPosition().IsKnown() {
 					if meta.Position.Line() == node.GetPosition().Line() {
+						// return error when node has both inilne code and children
```
Comment (cppforlife) [inline pkg/yamltemplate/metas.go]: let's remove this comment.

File: pkg/yamltemplate/metas.go [hunk 2]
```diff
@@ -61,6 +61,13 @@ func NewMetas(node yamlmeta.Node, opts MetasOpts) (Metas, error) {
...
+						if len(node.GetValues()) > 0 && node.GetValues()[0] != nil {
+							return metas, fmt.Errorf(
+								"Ambiguous code and value assignment at %s for node with children",
```
Comment (cppforlife) [inline pkg/yamltemplate/metas.go]: Expected YAML node at %s to have either computed or YAML value, but found both

**Comment by cppforlife** [comment]
@nimakaviani left a few comments to fix before merging.

**Comment by nimakaviani** [comment]
fixed.

**Pull request merged**